Gemfile.lock

```diff
@@ -164,20 +149,27 @@ GEM
       xcpretty-travis-formatter (>= 0.0.3)
     fastlane-plugin-appcenter (1.8.0)
     fastlane-plugin-sentry (1.6.0)
+    fastlane-plugin-wpmreleasetoolkit (1.0.1)
```

Comment:
We're at 1.2.0 now, probably worth updating on next cycle (also, did I update SNiOS to call `check_for_toolkit_update` already? I thought I did it in last release branch, maybe I did it in develop instead, worth a quick check for your next cycle.

Reply:
> did I update SNiOS to call check_for_toolkit_update already

It doesn't look like it: https://github.com/Automattic/simplenote-ios/pulls?q=is%3Apr+author%3AAliSoftware+is%3Aclosed

Reply:
Dammit indeed. Sorry, my bad; I'll let you add it (it's a one-liner in `before_all`) during your next code freeze if you want it.